topic_folders/policies/immediate-response-checklist.md

```diff
@@ -0,0 +1,74 @@
+## Carpentries Staff Immediate Response Checklist
+
+When a Code of Conduct (CoC) violation is found, one course of action is that activities in Carpentries channels (in-person and virtual) are monitored. In other cases, the community member is asked to take a termed suspension. When this is the resolution of the CoC committee, the following boundaries are enforced. The checklist below was created for incidents that require an immediate response, i.e. the community member is removed from all Carpentries engagement.
+
+The intended outcomes for such a suspension are immediate, ongoing, and demonstrable changes in any areas of engagement for which the CoC deemed undesirable in the incident review. 
+
```

Comment:
Brief comment: we need to separate between when issues have been resolved by the CoC, and issues that are under investigation. In the latter case, we need to ensure the confidentiality of both the reporter and the reportee, and there are responses listed below that would not maintain that during the investigative phase.

Reply:
I generally agree with this.  Having a distinction between being reported, under review, and resolved is important for privacy protection.  

That said, stating that the CoCc may choose to enforce some bans directly after a report or during the review process (but before they've made contact with the reported person) depending on the severity of the incident and if the behavior is active/continuing.  Imagine someone being actively harassed in github, list servs, or slack, they make a report, but the reported person isn't responding to CoCc members.  I feel like a report of something like that may reasonably trigger that user being banned from that platform until the review is resolved, but be a rare and specific type of situation.

Reply:
Suggestion for updated text:

As a part of their enforcement options, The Carpentries Code of Conduct (CoC) committee can place a community member in Termed Suspension, where the community member is removed from all Carpentries engagement, for a period of time.

The goal of a Termed Suspension is to 1) limit a person's participation in virtual and in-person Carpentries spaces for a set time to create a safe space for the community and 2) to give time and opportunity for the individual to review their response to the incident and demonstrate immediate, ongoing, and demonstrable changes as evaluated by the CoC committee before possible reinstatement.

In most cases a Termed Suspension will only be enacted after an incident has been reviewed by the CoC Committee and a conclusion reached. However, in severe situations, in particular where there is active harassment, the CoC Committee may enact an immediate short-term Termed Suspension while an incident is under review. 

If there is a decision for a Termed Suspension, the following checklist is followed by the appropriate personnel to enact the Termed Suspension. In accordance with CoC confidentiality guidelines, only essential personnel will be involved in implementation of these actions.

Reply:
This reads really well to me, and makes clear that those heavy handed actions during the review process are for extreme cases. This also reads as very flexible, where CoCc has a lot of power for discretion for a given incident.

Reply:
+1

Reply:
+1 from me too.